### PR TITLE
Add configurable webserver IP:PORT

### LINKED ISSUE #122: Change server HOST to default IP
Currently, if a user wants to access the webserver from another computer they need to navigate in lendingbot.py to the line where it says `HOST = "127.0.0.1"` and change it to `HOST = "0.0.0.0"`

Suggestion: Change this value to `0.0.0.0` by default

It makes the page accessible by localhost(aka `127.0.0.1`) AS WELL AS enabling routing to the page from another computer or router via the computer's local IP address.

I can't think of a reason not to do this.

### PR DESCRIPTION
**Pull request by Evanito**
## Description

The PR linked to #122 

Adds user-defined configuration to the `host:port` used by the bot's webserver.

## TESTING STAGE

Tested multiple use cases in config.
Documentation should handle proper usage.

Have not tested the arguments method, since I don't use it...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the code style of this project**
- [x] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [x] **If my change requires a change to the config, I have updated the config file and command line arguments accordingly.**
- [x] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

Allows users to access the page from LAN

Closes #122

### CODE REVIEW

File: lendingbot.py [hunk 1]
```diff
@@ -141,9 +142,9 @@
 	gapBottom = Decimal(config.get("BOT","gapbottom"))
 	gapTop = Decimal(config.get("BOT","gaptop"))
 	if(config.has_option("BOT", "sixtyDayThreshold")):
-                sixtyDayThreshold = float(config.get("BOT","sixtydaythreshold"))/100
-                xDayThreshold = sixtyDayThreshold
-                xDays = "60"
+				sixtyDayThreshold = float(config.get("BOT","sixtydaythreshold"))/100
```
Comment (rnevet) [inline lendingbot.py]: Is this indentation correct?

File: lendingbot.py [hunk 2]
```diff
@@ -344,9 +345,9 @@ def cancelAndLoanAll():
 		activeCurIndex += 1
 		activeCurTestBalance = Decimal(lendingBalances[activeCur])
 		activeBal = 0
-                if activeCur in totalLended:
-                	activeCurTestBalance += Decimal(totalLended[activeCur])
-		
+		if activeCur in totalLended:
+			activeCurTestBalance += Decimal(totalLended[activeCur])
```
Comment (Evanito) [inline lendingbot.py]: On this one I fixed the lines being unnecessarily indented. How it runs has not been changed

**Comment by Evanito** [comment]
@rnevet Can I get a LGTM?

**Comment by rnevet** [comment]
LWTM ;)

**Comment by Evanito** [comment]
LGTM

**Pull request merged**